crates/forge/tests/it/zk/paymaster.rs

```diff
@@ -0,0 +1,41 @@
+//! Forge tests for zksync contracts.
+
+use foundry_test_utils::util;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_zk_contract_paymaster() {
+    let (prj, mut cmd) = util::setup_forge(
+        "test_zk_contract_paymaster",
+        foundry_test_utils::foundry_compilers::PathStyle::Dapptools,
+    );
+    util::initialize(prj.root());
+
+    cmd.args([
+        "install",
+        "OpenZeppelin/openzeppelin-contracts",
+        "cyfrin/zksync-contracts",
+        "--no-commit",
+        "--shallow",
+    ])
+    .ensure_execute_success()
+    .expect("able to install dependencies");
+
+    cmd.forge_fuse();
+
+    let config = cmd.config();
+    prj.write_config(config);
+
+    prj.add_source("MyPaymaster.sol", include_str!("../../fixtures/zk/MyPaymaster.sol")).unwrap();
+    prj.add_source("Paymaster.t.sol", include_str!("../../fixtures/zk/Paymaster.t.sol")).unwrap();
+
+    cmd.args([
+        "test",
+        "--zk-startup",
+        "--evm-version",
```

Comment:
You should not need to hardcode evm version anymore

Reply:
We discussed this today with @Karrq, we thought we still needed to do it.

Reply:
Is there a reason you would still need this? IIRC this was before we added version sanitization because we were getting the invalid evm version error. That should not happen anymore.

Reply:
I thought I was getting error without it still